lib/adduser.js

```diff
@@ -21,20 +21,21 @@ function adduser (args, cb) {
     ))
   }
 
-  var registry = npm.config.get('registry')
-  var scope = npm.config.get('scope')
-  var creds = npm.config.getCredentialsByURI(npm.config.get('registry'))
+  const registry = npm.config.get('registry')
```

Comment:
You'll need `let registry` here, as it can be reassigned on L31 https://github.com/npm/cli/pull/190/files#diff-ca9f3ace6b674b53393b3d9678233b68R31